samples/sample-app.gradle

```diff
@@ -9,6 +9,24 @@ android {
         buildConfigField "String", "API_KEY", "\"${localProperties["apiKey"]}\""
         buildConfigField "String", "SITE_ID", "\"${localProperties["siteId"]}\""
     }
+    // Avoid redefining signing configs in sample apps to avoid breaking release
+    // builds (specially on CI servers)
+    signingConfigs {
+        debug {
+            storeFile file("${rootDir}/samples/debug.keystore")
+            storePassword 'android'
+            keyAlias 'androiddebugkey'
+            keyPassword 'android'
+        }
+    }
+    buildTypes {
+        debug {
+            signingConfig signingConfigs.debug
+        }
```

Comment:
Is there a reason we want to sign debug builds of the sample apps? I was expecting to only see code signing used for release builds.

Reply:
No, not really a specific reason. I just specified to avoid confusions as the default debug signingConfigs are from local machine, and new one we copied are also debug ones as we discussed on slack earlier. I'm okay with removing this for debug builds, but in that case, I'll suggest that we should rename the newly copied signing to release (or any other name than debug) to avoid any confusion.

Reply:
I agree with everything said here. I think it's OK to leave because debug code signing files are commonly known amongst Android devs and I don't see the configuration as it is in this PR causing confusion now or in the future.